android/gradle.properties

```diff
@@ -10,9 +10,12 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=4096m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
I'm not sure it's valid to have maxpermsize higher than mx? I'm building successfully with this `org.gradle.jvmargs=-Xmx2048M -Dkotlin.daemon.jvm.options\="-Xmx3072M" -XX\:MaxPermSize\=768m -XX\:+HeapDumpOnOutOfMemoryError -Dfile.encoding\=UTF-8` so maybe just going to 768 is okay

Reply:
Updated to `-XX:MaxPermSize=768m`